pkg/trace/test/testsuite/traces_test.go

```diff
@@ -87,6 +87,30 @@ func TestTraces(t *testing.T) {
 			payloadsEqual(t, append(p[:2], p[3:]...), v)
 		})
 	})
+
+	t.Run("filter_tags", func(t *testing.T) {
+		if err := r.RunAgent([]byte("apm_config:\r\n  filter_tags.require: [env:prod, db:mysql]\r\n  filter_tags.reject: [outcome:success]")); err != nil {
```

Comment:
It might be easier both to read and for you to write like this:
```suggestion
		if err := r.RunAgent([]byte(`apm_config:
	filter_tags
		require: ["env:prod", "db:mysq"l]
		reject: ["outcome:success"]`)); err != nil {
```
Otherwise it's getting confusing quickly. Use backticks, it makes it easier.